aioamqp/protocol.py

```diff
@@ -25,15 +25,15 @@ class AmqpProtocol(asyncio.StreamReaderProtocol):
     CHANNEL_FACTORY = amqp_channel.Channel
 
     def __init__(self, *args, **kwargs):
-        loop = kwargs.get('loop') or asyncio.get_event_loop()
-        super().__init__(asyncio.StreamReader(loop=loop), self.client_connected, loop=loop)
+        self._loop = kwargs.get('loop') or asyncio.get_event_loop()
```

Comment:
Since this attribute is used by other classes, shouldn't it be a "public" attribute, i.e ``self.loop``?